keycloak_backend/api/auth.py

```diff
@@ -21,6 +22,11 @@ def get_domain():
   return str(os.getenv('BACKEND_DOMAIN'))
 
 def get_credentials():
+  if os.getenv("KEYCLOAK_BACKEND_USER") is not None and os.getenv("KEYCLOAK_BACKEND_PASSWORD") is not None:
+    username = os.getenv("KEYCLOAK_BACKEND_USER")
+    password = generate_password_hash(os.getenv("KEYCLOAK_BACKEND_PASSWORD"))
+    return {username: password}
```

Comment:
Are the empty strings as bad as the `None` values in that case?

```suggestion
def get_credentials():
  user = os.getenv("KEYCLOAK_BACKEND_USER")
  raw_password = os.getenv("KEYCLOAK_BACKEND_PASSWORD")
  
  if user and raw_password:
      return {user: generate_password_hash(raw_password)}  
```